### PR TITLE
{vis}[intel/2017a] Blender v2.79, OpenImageIO v1.7.17, OpenEXR v2.2.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2158.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/02b30f8bf4d4dc39058aa717316c43a4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 4 (3 easyconfigs in this PR)
node2690.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/7da7f79448dccb81a5d15a41409f4038 for a full test report.

File: easybuild/easyconfigs/b/Blender/Blender-2.79-intel-2017a-Python-3.6.1.eb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    ('X11', '20170314'),
+    ('Mesa', '17.0.2'),
+    ('OpenImageIO', '1.7.17'),  # required for cycles render engine
+]
+
+builddependencies = [('CMake', '3.9.1')]
+
+separate_build_dir = 'True'
+
+modextravars = {'GALLIUM_DRIVER': 'swr'}
```
Comment (wpoely86) [inline easybuild/easyconfigs/b/Blender/Blender-2.79-intel-2017a-Python-3.6.1.eb]: I would add a comment here what this means: we use the intel software rasterizer by default (no gpu hardware acceleration)

File: easybuild/easyconfigs/o/OpenEXR/OpenEXR-2.2.0-intel-2017a.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+source_urls = ['https://github.com/openexr/openexr/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['8c219754af33fee199bdd72a0441f73dd4b3ce5363ac3c5cd7b2c6306f5442a6']
+
+builddependencies = [('CMake', '3.9.1')]
+
+env_vars = "CPATH=%(installdir)s/include/OpenEXR:$CPATH "
+env_vars += "LIBRARY_PATH=%(installdir)s/lib:$LIBRARY_PATH "
+env_vars += "LD_LIBRARY_PATH=%(installdir)s/lib:$LD_LIBRARY_PATH"
```
Comment (wpoely86) [inline easybuild/easyconfigs/o/OpenEXR/OpenEXR-2.2.0-intel-2017a.eb]: this needs the install path already included? what happens if you do a rebuild?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2690.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8dbeeecc6b80e330c9119cd23e2054c5 for a full test report.

File: easybuild/easyconfigs/o/OpenEXR/OpenEXR-2.2.0-intel-2017a.eb (hunk 2 above)
Comment (boegel) [reply easybuild/easyconfigs/o/OpenEXR/OpenEXR-2.2.0-intel-2017a.eb]: Rebuild works fine, just tested that.

Without this included it fails during the build step, you get errors like:

```
catastrophic error: cannot open source file "half.h"
```

EasyBuild performs two configure-build-install cycles (see list value for `preconfigopts`), and the `IlmBase` stuff must be available already when installing the `OpenEXR` stuff; when `IlmBase` is installed, the install directory is wiped if it existed already, and only when `OpenEXR` is being built are these env var updates put in place.

**Pull request merged**